idtoken/compute.go

```diff
@@ -23,6 +23,10 @@ func computeTokenSource(audience string, ds *internal.DialSettings) (oauth2.Toke
 	}
 	ts := computeIDTokenSource{
 		audience: audience,
+		format:   "full",
```

Comment:
This changes the default behaviour which is not desired.

Reply:
The default format was already full, so default behavior should be the same.